PowerEditor/installer/APIs/python.xml

```diff
@@ -30,6 +30,8 @@
 			<Overload retVal="" descr="Base class for warnings about bytes and buffer related problems, mostly&#x0a;related to conversion from str or comparing to str.">
 			</Overload>
 		</KeyWord>
+		<KeyWord name="case" />
+		<Keyword name="match" />
```

Comment:
It has to be in alphabetical order in order to work, isn't it?

Reply:
Yes.